pkg/testrunner/result/output_test.go

```diff
@@ -17,7 +17,7 @@ func TestValidationWebhook(t *testing.T) {
 }
 
 var _ = Describe("output generation tests", func() {
-
+	log := logger.Log.WithName("output_test")
```

Comment:
lets use the `NullLogger` if we do not need any log output in the unit tests?

Reply:
sure, done